src/main/resources/templates/recipe.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Recipe Page</title>
+    <style>
```

Comment:
I added in-line CSS so that the page looks good and isn't too basic. 

https://www.ucop.edu/electronic-accessibility/web-developers/advanced-tips/use-cascading-style-sheets-css-to-present-page-content.html#:~:text=Another%20advantage%20with%20CSS%20is,CSS%20are%20easier%20to%20maintain.